scripts/unicode.py

```diff
@@ -315,14 +315,14 @@ def gen_decomposition_tables(canon_decomp, compat_decomp, out):
     for table, name in tables:
         out.write("#[inline]\n")
         out.write("pub fn %s_fully_decomposed(c: char) -> Option<&'static [char]> {\n" % name)
-        out.write("    match c {\n")
+        out.write("    Some(match c {\n")
```

Comment:
@dotdash, can you add a comment here explaining the motivation for wrapping the match in `Some`?  otherwise, it may look non-idiomatic to future readers of this codegen script.